modules/book.js

```diff
@@ -0,0 +1,20 @@
+export default class Book {
+  constructor(title, author, index) {
+    this.title = title;
+    this.author = author;
+    this.index = index;
+  }
+
+  createNode() {
+    const Node = document.createElement('li');
+    const btn = document.createElement('button');
+
+    btn.textContent = 'Remove';
+    Node.innerHTML = `
+      <span>${this.title} by ${this.author}</span>
+    `;
+    Node.appendChild(btn);
+
+    return { Node, btn, index: this.index };
+  }
```

Comment:
- [ ] 🙏Please use arrow functions for all of your class methods. This is a project requirement and the recommended👍 JavaScript ES6 best practice. Check out the below project requirements:
- Refactor your functions to arrow functions.

Reply:
> Correction

The methods have been restructured to arrow function.